include/seqan/basic/alphabet_simple_type.h

```diff
@@ -319,7 +380,7 @@ struct MinValue<SimpleType<TValue, TSpec> >
 };
 
 template <typename TValue, typename TSpec>
-const SimpleType<TValue, TSpec> MinValue<SimpleType<TValue, TSpec> >::VALUE = SimpleType<TValue, TSpec>(0);
+const SimpleType<TValue, TSpec> MinValue<SimpleType<TValue, TSpec> >::VALUE = std::numeric_limits<SimpleType<TValue, TSpec>>::min();
```

Comment:
when removing the MIn/MaxValue specialization for SimpleType I end up with the following error:

```
/home/mi/smehringer/AGABI/seqan/include/seqan/basic/alphabet_math.h:78:49: error: ambiguous overload for ‘operator<<’ (operand types are ‘seqan::SimpleType<unsigned char, seqan::Dna_>’ and ‘int’)
 const T MaximumValueSigned_<T>::VALUE = ((((T)1 << (BitsPerValue<T>::VALUE - 2)) - 1) << 1) + 1;
```
I don't quite understand what is the problem here.